rest-api/src/main/java/gov/cms/qpp/conversion/api/services/StorageServiceImpl.java

```diff
@@ -107,19 +112,20 @@ public InputStream getFileByLocationId(String fileLocationId) {
 	 * @return file used for cpc+ validation.
 	 */
 	@Override
-	public InputStream getCpcPlusValidationFile() {
-		String bucketName = environment.getProperty(Constants.CPC_PLUS_BUCKET_NAME_VARIABLE);
-		String key = environment.getProperty(Constants.CPC_PLUS_FILENAME_VARIABLE);
-		if (StringUtils.isEmpty(bucketName) || StringUtils.isEmpty(key)) {
-			API_LOG.warn("No CPC+ bucket name and/or CPC+ key specified");
+	public byte[] getCpcPlusValidationFile() {
+		if (StringUtils.isEmpty(Constants.FMS_TOKEN)) {
+			API_LOG.warn("No FMS_TOKEN specified");
 			return null;
 		}
-		API_LOG.info("Retrieving CPC+ validation file from bucket {}", bucketName);
-
-		GetObjectRequest getObjectRequest = new GetObjectRequest(bucketName, key);
-		S3Object s3Object = amazonS3.getObject(getObjectRequest);
-
-		return s3Object.getObjectContent();
+		API_LOG.info("Retrieving CPC+ validation file from FMS");
+
+		RestTemplate rest = new RestTemplate();
+		RequestEntity<?> entity = RequestEntity.get(
+					URI.create(Constants.AR_API_BASE_URL + "/v1/fms/file/qppct/cpc_plus_validations"))
```

Comment:
Does this retrieve all files in the `/v1/fms/file/qppct/cpc_plus_validations` directory in FMS?

Reply:
Fixed for clarity